lib/mayaUsd/render/MaterialXGenOgsXml/CMakeLists.txt

```diff
@@ -37,6 +37,17 @@ target_sources(${PROJECT_NAME}
     )
 endif()
 
+if(USD_HAS_BACKPORTED_MX39_OPENPBR)
```

Comment:
The backported OpenPBR code requires injecting Maya lighting code at the exact right moment. This is done by creating a special closure node to handle to handle the two MaterialX nodes that require updated lighting.